internal/migrations/20201202140700_change_image_ssh_key_text_test.go

```diff
@@ -45,26 +47,28 @@ var _ = Describe("changeOverridesToText", func() {
 	})
 
 	It("Migrates down and up", func() {
-		t, err := getColumnType(db, &common.Cluster{}, "image_ssh_public_key")
-		Expect(err).ToNot(HaveOccurred())
-		Expect(t).To(Equal("TEXT"))
+		if db.Migrator().HasColumn(&common.Cluster{}, "image_ssh_public_key") {
```

Comment:
Why add this? Pretty sure the column still exists.

Reply:
Will remove